### DISCUSSION TITLE
Ekuiper rules sending notifications service

### DISCUSSION THREAD

**Discussion by antiadominguez** [edgexfoundry/community#285] [Q&A]
I see on the documentation that you can configure ekuiper rules engine to send notifications with the notifications service, but I don't know how to do it and I didn't find anything on the documentation. I tryed with a rest sink but I didn't send the notification. How do I configure the sink to send a email notification with the service.

**Comment by cloudxxx8** [comment]
you may move this question to the equiper community

**Comment by jiekechoo** [comment]
@antiadominguez According to the eKuiper docs, there is no `Email` sink in the `Sink Connectors`, you can find the details in [here](https://ekuiper.org/docs/en/latest/guide/sinks/overview.html). 

If you just want to send an Email notification, you can try [this doc](https://docs.edgexfoundry.org/3.1/microservices/support/notifications/HowTo_EmailNotifications/).
